itwin-platform-access/imodels-access-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/imodels-access-common",
-  "version": "5.2.3",
+  "version": "5.2.4",
```

Comment:
would recommend creating a 6.0.0-dev pre-release that publishes esm ver, and change cjs path from lib to lib/cjs

Reply:
the change to reexport `AccessTokenAdapter` from barrel instead of absolute path does not break anything, but we can push it to 6.0.0-dev to be on the safe side. 

I don't want any import path to be specific with `lib/cjs` as that will force to have 2 versions of source files both for esm and cjs, resolve them during build time. Import from barrel file solves this problem nicely, because you indicate only the package name in the import clause without telling anything about its type. It's task for the bundler then to resolve dependencies of proper flavour, either all esm or commonjs

Reply:
I think the idea has been to hide as many internal implementation details as possible and only publicly export parts that are absolutely necessary. No barrel file likely meant that you're not supposed to import from imodel-access-common and only import what's reexported from other packages, which seems like overly restrictive policy.

Reply:
And we are planning to start a new major version changes, so I think it would be safer to mark this as a dev version for that.